### PR TITLE
code review changes

### PR DESCRIPTION
**Pull request by jbarr-code**

### CODE REVIEW

**Comment by jbarr-code** [comment]
Analysis:
This program is used to keep track of the materials and time need to complete a 3D print. It utilizes RoomDB to store all this data. The program was available on time for me and it does compile. 

For the rationale behind my changes I found that the Id's for the database were not written using camel case so it was confusing to read. The formatting for the modelDTO had the curly brace on a different line that all the other dto's so it was more confusing to read. Finally, I added the const variable to the BASE_URL as the base url will not change and stay constant. 

Three commits:
https://github.com/sekhonsj/CalendarTech/commit/8f85235e6614c8b95468f18e1b932125a02cbb5e
https://github.com/sekhonsj/CalendarTech/commit/37586e08ae40c62ada3abfcc230af1a092cf5c57
https://github.com/sekhonsj/CalendarTech/commit/b0d0e6e66ee3a952a1f854c62bd08db1637c7b1d

1. I learned how to connect my dot variables directly to the database column names.

data class Model (@PrimaryKey var mId: Int,
                  @ColumnInfo(name = "modelName")var modelName: String,
                  @ColumnInfo(name = "artistId")var artistId: Int,
                  @ColumnInfo(name = "modelUrl")var url: String)
{

2. I learned how to write a database query within Kotlin

@Query("SELECT * FROM Printer WHERE printerName LIKE :name")
    fun findByName(name: String): Printer

3. I learned how to implement classes into creating a database.

@Database(entities = arrayOf(Brand::class, Material::class, Model::class, Printer::class), version=1)
abstract class SliceDatabase : RoomDatabase() {
    abstract fun printerDao(): IPrinterDAO
//    abstract fun materialDao(): IMaterialDAO
    abstract fun modelDao(): IModelDAO
    abstract fun brandDao(): IBrandDAO
}

File: app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt [hunk 1]
```diff
@@ -5,11 +5,11 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 /**
- * @param bid: DB ID for dev use
+ * @param bId: DB ID for dev use
  * @param brandName: User Inputted Name
  */
 @Entity(tableName = "Brand")
-data class Brand(@PrimaryKey val bid: Int,
+data class Brand(@PrimaryKey val bId: Int,
```
Comment (jbarr-code) [inline app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt:12]: Made the variable camel case as it is easier to read and follows Kotlin best practices. https://kotlinlang.org/docs/coding-conventions.html#function-names

This is repeated over many files for the id's but I only left the comment on this one.

File: app/src/main/java/edu/uc/kovaciad/slicetracker/RetrofitInstance.kt [hunk 2]
```diff
@@ -5,7 +5,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "127.0.0" // TODO: Replace with our webapp (Sprint 2)
+    private const val BASE_URL = "127.0.0" // TODO: Replace with our webapp (Sprint 2)
```
Comment (jbarr-code) [inline app/src/main/java/edu/uc/kovaciad/slicetracker/RetrofitInstance.kt:8]: Added const as the base url will not change. https://kotlinlang.org/docs/coding-conventions.html#property-names

File: app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Model.kt [hunk 3]
```diff
@@ -5,16 +5,17 @@ import androidx.room.Entity
...
  * @param modelUrl: model URL
  */
 @Entity(tableName = "Model")
-data class Model (@PrimaryKey var mid: Int,
+data class Model (@PrimaryKey var mId: Int,
                   @ColumnInfo(name = "modelName")var modelName: String,
                   @ColumnInfo(name = "artistId")var artistId: Int,
-                  @ColumnInfo(name = "modelUrl")var url: String) {
+                  @ColumnInfo(name = "modelUrl")var url: String)
+{
```
Comment (jbarr-code) [inline app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Model.kt:18]: Moved to curly brace to the next line as all other dto's had it on this line. Moved it for consistency and readability.

File: app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt:12]: A good idea to align with conventions.  In this case, I'd spell it out entirely.  What is a bId?  I don't know.  But a brandId does make more sense.

As tempting as it is to save keystrokes with abbreviations, you'll find that you waste more time trying to remember which abbreviation you used, than the time you saved on keystrokes with a shorter name.  :)

File: app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Material.kt [hunk 4]
```diff
@@ -6,15 +6,15 @@ import androidx.room.Entity
...
+ * @param mId: DB ID for dev use
  * @param materialName: User Inputted Name
  * @param materialTypeId: Type of material from predetermined list
  * @param brandId: Brand from brand object
  * @param materialColor: Color of filament. Uses android color ints
  * @param materialAmount: Amount of filament in g or L
  */
 @Entity(tableName = "material")
-data class Material (@PrimaryKey var mid: Int,
+data class Material (@PrimaryKey var mId: Int,
```
Comment (discospiff) [inline app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Material.kt:17]: Ditto comment above.  As long as you're in here, name it materialId.  materialID is also acceptable, as there's not a well defined convention for acronyms.

File: app/src/main/java/edu/uc/kovaciad/slicetracker/dto/MaterialType.kt [hunk 5]
```diff
@@ -5,11 +5,11 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 /**
- * @param mtid: DB ID for dev use
+ * @param mtId: DB ID for dev use
  * @param materialTypeName: User Inputted Name
  */
 @Entity(tableName = "MaterialType")
-data class MaterialType(@PrimaryKey val mtid : Int,
+data class MaterialType(@PrimaryKey val mtId : Int,
```
Comment (discospiff) [inline app/src/main/java/edu/uc/kovaciad/slicetracker/dto/MaterialType.kt:12]: Ditto above.  materialTypeId or materialTypeID.

**Comment by discospiff** [comment]
In lieu of merging this branch, I'd go one step further and spell out the names, so that the variables are self-describing.

**Pull request closed without merging**